game/src/ecs/entities/Entity.java

```diff
@@ -14,10 +15,13 @@ public class Entity {
     private static int nextId = 0;
     public final int id = nextId++;
     private HashMap<Class, Component> components;
+    private final Logger entityLogger = Logger.getLogger(this.getClass().getName());
 
     public Entity() {
         components = new HashMap<>();
         Game.entities.add(this);
+        Logger entityLogger = Logger.getLogger(this.getClass().getName());
```

Comment:
Warum wird hier ein neuer Logger erstellt? Es wird doch bereits einer angelegt als Attribut.

Reply:
Ja, das kam durch das murkse rebasen und mein Übersehen
fixed in 5b1bab7be907bb89b3f5a5e141fdb54829135c10